### PR TITLE
Link browser tests

### PR DESCRIPTION
**Pull request by NSeydoux**
This PR enforces that browser-based tests run on the local code, rather than on the latest published version.

- [X] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: e2e/browser/test/playwright.config.ts [hunk 1]
```diff
@@ -24,14 +24,22 @@ import { PlaywrightTestConfig } from "@playwright/test";
...
   use: {
-    baseURL: "http://localhost:1234",
+    baseURL: "http://localhost:3000",
     headless: true,
     screenshot: "only-on-failure",
     trace: "on",
     video: "on-first-retry",
   },
+  webServer: {
+    command: "cd ../src/ ; npm run dev",
```
Comment (matthieubosquet) [inline e2e/browser/test/playwright.config.ts:38]: Not sure what `npm run dev` refers to?

**Review by matthieubosquet — APPROVED** [review]
Admittedly not the best for reviewing this kind of PR, but it looks good.

I'm not sure what the `npm run dev` command points to (not in package.json's scripts).

File: e2e/browser/test/playwright.config.ts (hunk 1 above)
Comment (ThisIsMissEm) [reply e2e/browser/test/playwright.config.ts:38]: Ah, I think I may have removed it recently, it was running the e2e browser app.

Comment (matthieubosquet) [reply e2e/browser/test/playwright.config.ts:38]: hahaha, lol, it's a mystery why it would run then 🤔 ?

Comment (NSeydoux) [reply e2e/browser/test/playwright.config.ts:38]: That's the command to run the NextJS server

Comment (matthieubosquet) [reply e2e/browser/test/playwright.config.ts:38]: Nice! Good to know :)

**Pull request merged**